pkg/v1/tkg/kappclient/client.go

```diff
@@ -36,7 +36,7 @@ func (c *client) GetClient() crtclient.Client {
 }
 
 // NewKappClient returns a new kapp client
-func NewKappClient(kubeCfgPath string) (Client, error) {
+func NewKappClient(kubeCfgPath, kubeContext string) (Client, error) {
```

Comment:
suggestion: this is completely optional, i think we can rename this function to NewKappClientForContext and add a new function NewKappClient and make a call to NewKappClientForContext, that way it would be much cleaner, and you can make changes only where needed, something like below:

```
func NewKappClient(kubeCfgPath string) {
     return NewKappClientForContext(kubeCfgPath, "")
}
```

you can also do something similar for GetKubeConfig and NewTKGPackageClient functions.

Reply:
Thanks. Makes complete sense. I have updated the PR based on this suggestion.